client/src/components/atoms/style/BannerCarousel.scss

```diff
@@ -0,0 +1,39 @@
+.banner-carousel {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    align-items: center;
+
+    .carousel {
+        z-index: 0;
+        display: block;
+
+        img {
+            width: 100%;
+            opacity: 0.4;
+            height: 435px;
+            object-fit: cover;
+        }
+    }
+
+    &-info {
```

Comment:
width of text overlay needs to a bit larger

figma:
![Screenshot from 2021-07-26 20-30-28](https://user-images.githubusercontent.com/20423590/127076199-8960c3a8-2929-40a7-b6ec-efbb8a26fa20.png)


current implementation at 1440px:
![Screenshot from 2021-07-26 20-28-56](https://user-images.githubusercontent.com/20423590/127076163-312b6fa9-2be4-45f5-814b-eca88b671e3f.png)